paymentsheet-example/src/main/java/com/stripe/android/paymentsheet/example/playground/PaymentSheetPlaygroundViewModel.kt

```diff
@@ -299,6 +304,66 @@ internal class PaymentSheetPlaygroundViewModel(
         status.value = StatusMessage(statusMessage)
     }
 
+    @OptIn(ExperimentalCustomerSheetApi::class)
+    fun fetchOption(customerSheet: CustomerSheet) {
```

Comment:
As is, there could be multiple requests at the same time. If we have recomposition issues, this could be really bad.